tree/ntuple/v7/doc/specifications.md

```diff
@@ -338,13 +338,14 @@ The flags field can have one of the following bits set
 
 The structural role of the field can have on of the following values
 
-| Value    | Structural role                                          |
-|----------|----------------------------------------------------------|
-| 0x00     | Leaf field in the schema tree                            |
-| 0x01     | The field is the mother of a collection (e.g., a vector) |
-| 0x02     | The field is the mother of a record (e.g., a struct)     |
-| 0x03     | The field is the mother of a variant (e.g., a union)     |
-| 0x04     | The field is a reference (pointer), TODO                 |
+| Value    | Structural role                                                          |
+|----------|--------------------------------------------------------------------------|
+| 0x00     | Leaf field in the schema tree                                            |
+| 0x01     | The field is the mother of a collection (e.g., a vector)                 |
+| 0x02     | The field is the mother of a record (e.g., a struct)                     |
+| 0x03     | The field is the mother of a variant (e.g., a union)                     |
+| 0x04     | The field is a reference (pointer), TODO                                 |
```

Comment:
Is this still a todo or it is 'replaced` by `0x05` per se?

Reply:
This is not clear yet. We may keep the 0x04 number reserved in the 1.0 spec.

Reply:
Ah, I guess you mean we should then give the `unsplit` field type the id 0x04. Done.